src/_posts/compliance/2000-01-01-security-overview.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
To me it's a bit weird to have `Security` and `Compliance` in the navigation, then in `Compliance` having `Security Overview`, I've the feeling it's not really intuitive

Reply:
The reasoning was : it's always for compliance reasons that you want to have a security overview

The "security" section is very operational, whereas the overview is all about rules and theory

Would you prefer having an "overview" section under "security" ?

Reply:
I don't know what's best it was a bit disturbing, maybe it's not `Security Overview` but `Policies Overview` or `Trust Center` as we discussed it. I don't know actually